configs/bonitasoft.json

```
@@ -20,4 +53,4 @@
     "1393529298"
   ],
```

```suggestion
  ],
  "custom_settings": {
    "attributesForFaceting": ["version", "tags"]
  },
```

This will retrieve the `version` and `tags` and allow you to use them as `facetFilters`